src/app/layout.config.tsx

```
@@ -1,33 +1,58 @@
+import Logo from '@/public/logo.png';
```

ESLint

[error] 'Logo' is defined but never used.

(@typescript-eslint/no-unused-vars)